ext/opcache/tests/packages_invalidation.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Files are invalidated if the package definition changes
+--SKIPIF--
+<?php require_once('skipif.inc'); ?>
+--CONFLICTS--
+server
+--FILE--
+<?php
+
+include __DIR__ . "/php_cli_server.inc";
+php_cli_server_start(getenv('TEST_PHP_EXTRA_ARGS'));
+
+echo file_get_contents("http://" . PHP_CLI_SERVER_ADDRESS . "/packages_invalidation_with_strict_types.php" );
```

Comment:
Hello, worth noting that tests/*.php files are gitignored... Some editors, and IDEs have issue with this and recommend unignoring theses. And for possible automated tools it is simpler to not add ignored files in the repo. So, maybe a simple rename of */tests/*.php -> */tests/*.inc could be a solution.

Reply:
These tests use the built-in server, which doesn't support `.inc` as a file-extension for PHP files.

Reply:
Then maybe we can invent a better file extensions naming strategy here? For example, either tests output files would be simple `.php` (a bit misnamed strategy from the beginning but let's have it now), and such "code" or included files would be `.inc.php` or something? That way they can be ignored and tracked like this in gitignore file:
```gitignore
**/tests/**/*.php
!**/tests/**/*.inc.php
```